src/pykoala/spectra/onedspec.py

```diff
@@ -0,0 +1,3015 @@
+                5191.82, #5517.71,   
+                #red  #  [OI]   [SIII]  [OI]   Ha+[NII]  HeI    [SII]     HeI   [ArIII]  [OII]  [ArIII]    P12 P11 P10    [SIII]         
+                6300.3, 6312.10, 6363.78, 6548.03, 6562.82, 6583.41, 6678.15, 
+                6716.47, 6730.85, 7065.28, 7135.78, 7318.39, 7329.66, 7751.1, 
+                8750.47, 8862.79, 9014.91,
+                9068.9] 
+
+el_list_names = ["[OII]", "[OII]", 
+                 "[NeIII]",  "H8",  "HeI", "[NeIII]",  "H7", # "HeI+[NeIII]"
+                 "[SII]", #"[SII]",
+                 "Hd","Hg", "[OIII]", 
+                 #"HeI",
+                 "HeI", "[FeIII]",
+                 "HeII", "Hb", "[OIII]", "[OIII]",
+                 "[ArIII]", # "[ClIII]",
+                 #red
+                "[OI]", "[SIII]", "[OI]","[NII]","Ha","[NII]","HeI","[SII]","[SII]","HeI","[ArIII]","[OII]","[OII]","[ArIII]",
+                 "P12", "P11", "P10",
+                 "[SIII]"]
+el_list_fl = [0.322, 0.322,   
+              0.291, 0.286,   0.267, 0.267, 0.266,
+              0.239, #0.237,
+              0.230, 0.157, 0.149,
+              #0.126 ,
+              0.115,  0.058,             
+              0.050,  0.000, -0.032, -0.038,
+              -0.081, #-0.145 ,
+              #red  
+              -0.263, -0.264, -0.271, -0.296,  -0.298,  -0.300, -0.313,  -0.318, -0.320, -0.364, -0.374, -0.398, -0.400, -0.455,
+              -0.568,-0.578 , -0.590,
+              -0.594]
+
+
+
+# EL  dictionary 
+emission_line_dictionary = {}
+
+for i in range(len(el_list_no_z)):
+    emission_line_dictionary[el_list_no_z[i]] = [el_list_names[i], el_list_fl[i]]
+
+#  [OI]   [SIII]  [OI]   Ha+[NII]  HeI    [SII]     HeI   [ArIII]  [OII]  [ArIII]  [SIII]  
+# el_low_list_no_z  =[6296.3, 6308.1, 6359.8, 6544.0, 6674.2, 6712.5, 7061.3, 
+#                     7129., 7312., 7747.1,   0,0,0,   9063.9]                     ### Careful with 0,0,0 as extra added to el_list
+# el_high_list_no_z =[6304.3, 6316.1, 6367.8, 6590.0, 6682.2, 6736.9, 7069.3, 
+#                     7141., 7336., 7755.1,   0,0,0,   9073.9]
+
+# =============================================================================
+# Sky lines
+# =============================================================================
+
+
+# These are for KOALA, I should move them there....
+
+edge_fibres_100 = [ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15, 16,
+       17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33,
+       34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49,
+       936, 937, 938, 939, 940, 941, 942, 943, 944, 945, 946, 947, 948, 949, 950, 951,
+       952, 953, 954, 955, 956, 957, 958, 959, 960, 961, 962, 963, 964,
+       965, 966, 967, 968, 969, 970, 971, 972, 973, 974, 975, 976, 977,
+       978, 979, 980, 981, 982, 983, 984, 985]
+
+fibres_best_sky_100 = [ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15, 16,
```

Comment:
Same as above. Here I wonder whether all instruments may indeed have a predefined set of "sky fibres". As a matter of fact, WEAVE does, but I wasn't aware that KOALA did. Or are you using a set similar to the "edge fibres", assuming that they are probably dominated by sky. What if they aren't (as it is often the case in HI-KIDS)?